web_external/js/views/body/MapPanel.js

```diff
@@ -214,9 +215,10 @@ minerva.views.MapPanel = minerva.View.extend({
                     }
                 })
             });
-            this.map.createLayer(this.session.sessionJsonContents.basemap);
+            this.map.createLayer(this.session.sessionJsonContents.basemap,
```

Comment:
make an API on the session model rather than mucking with the sessionJsonContents directly